README_en.md

```diff
@@ -21,467 +21,97 @@ PaddleClas is a toolset for image classification tasks prepared for the industry
+    - [Mobile Deployment](./deploy/lite/readme.md)
+    - [Inference Using whl](./docs/zh_CN/whl.md)
+- Advanced Tutorial
+    - [Knowledge Distillation](./docs/zh_CN/advanced_tutorials/distillation/distillation.md)
+    - [Model Quantization](./docs/zh_CN/extension/paddle_quantization.md)
+    - [Data Augmentation](./docs/zh_CN/advanced_tutorials/image_augmentation/ImageAugment.md)
+- FAQ(Suspended Updates)
+    - [Image Classification FAQ](docs/zh_CN/faq.md)
+- [License](#许可证书)
+- [Contribution](#贡献代码)
+
+
+## Introduction to Image Recognition Systems
+
+<a name="Introduction to Image Recognition Systems"></a>
+<div align="center">
+<img src="./docs/images/structure.png"  width = "400" />
+</div>
 
-| Model                     | Top-1 Acc | Top-5 Acc | time(ms)<br>bs=1 | time(ms)<br>bs=4 | Flops(G) | Params(M) | Download Address |
-|------------------------|-----------|-----------|------------------|------------------|----------|-----------|------------------------------------------------------------------------------------------------------|
-| AlexNet       | 0.567 | 0.792 | 1.44993         | 2.46696         | 1.370 | 61.090 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/AlexNet_pretrained.pdparams) |
-| SqueezeNet1_0 | 0.596 | 0.817 | 0.96736 | 2.53221         | 1.550 | 1.240 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/SqueezeNet1_0_pretrained.pdparams) |
-| SqueezeNet1_1 | 0.601 | 0.819 | 0.76032       | 1.877      | 0.690   | 1.230 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/SqueezeNet1_1_pretrained.pdparams) |
-| VGG11 | 0.693 | 0.891 | 3.90412 | 9.51147 | 15.090 | 132.850 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG11_pretrained.pdparams) |
-| VGG13 | 0.700 | 0.894 | 4.64684 | 12.61558 | 22.480 | 133.030 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG13_pretrained.pdparams) |
-| VGG16 | 0.720 | 0.907 | 5.61769 | 16.40064 | 30.810 | 138.340 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG16_pretrained.pdparams) |
-| VGG19 | 0.726 | 0.909 | 6.65221 | 20.4334 | 39.130 | 143.650 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/VGG19_pretrained.pdparams) |
-| DarkNet53 | 0.780 | 0.941 | 4.10829 | 12.1714 | 18.580 | 41.600 | [Download link](https://paddle-imagenet-models-name.bj.bcebos.com/dygraph/DarkNet53_pretrained.pdparams) |
+The steps of image recognition can be divided into three steps:
+- （1）Identify region proposal for image objects through a detection model；
+ - （2）Extract the image features for each region proposal;
+- （3）Match the extracted features with the images in the retrieval database and output results;
 
+For a new unknown category, there is no need to retrain the model, just fill in the retrieval database with images of that category, recreate the retrieval database, and the category can be recognised.
 
```

Comment:
just prepare images of new category, extract features and update retrieval database